### PR TITLE
Clean up files generated by node agent unit tests.

### PR DESCRIPTION
**Pull request by myidpt**

### CODE REVIEW

File: security/cmd/node_agent/na/nodeagent_test.go [hunk 1]
```diff
@@ -47,6 +48,9 @@ func TestStartWithArgs(t *testing.T) {
 	}
 	signedCert := []byte(`TESTCERT`)
 	certChain := []byte(`CERTCHAIN`)
+	defer os.Remove(generalConfig.CAClientConfig.RootCertFile)
```
Comment (incfly) [inline security/cmd/node_agent/na/nodeagent_test.go]: Optional, put all os.Remove in as single defer func() {...}()?

Comment (myidpt) [reply security/cmd/node_agent/na/nodeagent_test.go]: Sure.

**Comment by myidpt** [comment]
@incfly I think you may need to approve by clicking on "approve" in "review changes".

**Comment by incfly** [comment]
Because I'm not OWNER yet? Could you approve https://github.com/istio/istio/pull/4714?

**Comment by myidpt** [comment]
I see. Add /lgtm to that PR.

**Pull request merged**